### ISSUE TITLE
Tactical Screen doesn't show frequency options

### ISSUE THREAD

**Issue by Corsaka** [daid/EmptyEpsilon#551]
The tactical console doesn't show any beam/shield frequencies, meaning it's incredibly difficult to take out an enemy when you have less players. A simple workaround is selecting helm and weapons on 6/5 player mode and swapping between them, but that's very annoying to do and can often slow reaction times to the point where it's impossible to win a fight.

**Comment by Corsaka** [comment]
Operations screen doesn't show other important things such as hacking and relay view either.

**Comment by amir-arad** [comment]
I think it's by design. smaller crew - less capabilities

**Comment by daid** [comment]
The hacking and relay is by design (AKA, they are not essential), the frequencies is not, that's by mistake.

**Comment by NinjaSpectre** [comment]
In my customized build I gave the frequency selectors to the Engineering Advanced screen.  I figured that was the best place, otherwise the only difference between Engineering and Engineering+ is raising/lowering shields.  Downside is that there is not much screen real estate there.

**Comment by amir-arad** [comment]
PR?

**Comment by NinjaSpectre** [comment]
It is not a simple change since I had to re-arrange some of the Engineering screen layout as well.  Mainly because I needed more useful space where the custom ship functions are currently located.
I would greatly appreciate another developer/modder insight.  I will clean up what I have and post something in this topic when I get some time. (might take a day or two to gather only the relevant changes)

**Comment by NinjaSpectre** [comment]
I made a branch [containing my changes.](https://github.com/ninjaspectre/EmptyEpsilon/tree/crew-4-screen-update)  Wasn't quite as bad as I had thought once I cleaned it up.

[Commit 4471bb6](https://github.com/NinjaSpectre/EmptyEpsilon/commit/4471bb61aa2c32a695b745626d707d46da3662e6) is the addition of frequency selectors and the moving of custom functions in Engineering+.  I think the screen could be better if a good place could be found for the self destruct.

[Commit c032e02](https://github.com/NinjaSpectre/EmptyEpsilon/commit/c032e02e6cd182b6741fd32d8193486f17e4a08a) adds the alert level selector and a reputation display to the Operations screen.
